packages/contracts-versions/commit_hashes.json

```diff
@@ -1,7 +1,6 @@
 {
   "versions": {
-    "v0_7_0_alpha": "9b912728c315f983af8c4bfd1f8d07021996563f",
-    "v1_0_0_mainnet_goerli": "c2b9d23a96654e81f22fbf91e6f334ef26a370af",
-    "v1_0_0_mumbai": "9485d97301611cfc78faa4bd00eb54abb6dd2d5e"
+    "v1_0_1": "c2b9d23a96654e81f22fbf91e6f334ef26a370af",
```

Comment:
No v1_0_0?

Reply:
i just picked up from here https://www.npmjs.com/package/@aragon/osx?activeTab=versions
do u know the commit hash of 1_0_0? so we can add it